dataactvalidator/filestreaming/fieldCleaner.py

```diff
@@ -111,10 +111,17 @@ def cleanRow(row, fileType, validationInterface):
         for key in row.keys():
             field_type = validationInterface.getColumn(key, fileType).field_type.name
             value = row[key]
-            if value is not None and field_type in ["INT", "DECIMAL", "LONG"]:
-                tempValue = value.replace(",","")
-                if FieldCleaner.isNumeric(tempValue):
-                    row[key] = tempValue
+            if value is not None:
+                # Remove extra whitespace
+                value = value.strip()
+                if field_type in ["INT", "DECIMAL", "LONG"]:
+                    tempValue = value.replace(",","")
```

Comment:
Now that we're stripping commas here, does that mean the code @nmonga91 added in the validation routine to ignore commas is no longer needed?